mica-webapp/src/main/resources/_templates/datasets.ftl

```diff
@@ -5,12 +5,15 @@
 <!-- Template variables -->
 <#if !type??>
   <#assign title = "datasets">
+  <#assign className = "Study,HarmonizationStudy">
```

Comment:
Why not using the "type" instead of introducing "className" that has the same semantic?
(the idea is to minimize the template changes)

Reply:
This is because it will be used with `Mica_study.className` such that it will be reflected in the widget in the search page.

Reply:
ok but do not assign className in the page's main template: minimize the changes so that users do not suffer (too much) when upgrading.